app/build.gradle

```diff
@@ -129,6 +129,7 @@ dependencies {
     implementation "androidx.fragment:fragment:$fragmentVersion"
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
     implementation "androidx.media:media:$mediaVersion"
+    implementation 'com.android.support:multidex:2.0.1'
```

Comment:
This should be `androidx.multidex:multidex:2.0.1`

https://mvnrepository.com/artifact/androidx.multidex/multidex

Reply:
Oh, right. Thanks for reviewing my PRs!

Reply:
No problem :)